server/src/models/Show.js

```diff
@@ -0,0 +1,20 @@
+const Model = require("./Model.js")
+
+class Show extends Model {
+  static get tableName() {
+    return "shows"
+  }
+
+  static get jsonSchema() {
+    return {
+      type: "object",
+      required: ["name"],
+      properties: {
+        name: {type: "string", minLength: 1, maxLength: 255},
+        description: {type: "string", minLength: 10}
+      }
```

Comment:
i wouldnt reject for this, but ideally there is a space after the first curly bracket and before the last curly bracket in a object